CaesarCipher/CaesarCipher/main.m

```diff
@@ -39,22 +72,79 @@ - (NSString *)encode:(NSString *)string offset:(int)offset {
         char *e = strchr(allchars, buffer[i]);
         int idx= (int)(e - allchars);
         int new_idx = (idx + offset) % strlen(allchars);
-
+        
         result[i] = allchars[new_idx];
     }
-
+    
     return [NSString stringWithCharacters:result length:count];
 }
 
+
+
 - (NSString *)decode:(NSString *)string offset:(int)offset {
     return [self encode:string offset: (26 - offset)];
 }
 
+
+// Write a method called codeBreaker which accepts two cipher strings as paramaters and returns a boolean value which tells us whether they are actually the same input message encoded using two different offsets
+
+
+- (BOOL) codeBreaker:(NSString *)string1 cipherTwo:(NSString *)string2 {
+    
+    for (int a = 0; a < string1.length; a++) {      // index starts at 0; if a is less than string 1 length, increment by 1
```

Comment:
string1.length isn't going to loop through all possible offsets of a string (25).

```objective-c
for (int a = 0; a < 26; a++) {
```